src/CodeGeneration.Roslyn.Tests.Generators/AddGeneratedExternGenerator.cs

```diff
@@ -26,7 +26,7 @@ public class AddGeneratedExternGenerator : RichBaseGenerator
         public AddGeneratedExternGenerator(AttributeData attributeData)
             : base(attributeData)
         {
-            Extern = (string)AttributeData.ConstructorArguments[0].Value;
+            Extern = (string)AttributeData.ConstructorArguments[0].Value!;
```

Comment:
I wonder if this non-null assumption is wise. Even if the attribute's own ctor checks for null, we're not accessing the value after it passes through the ctor -- we're accessing the argument before it's passed in. The compiler hasn't even run yet, so I don't think anything stops the user from passing null in and then we'd dereference that.
Should we check and throw instead?

Reply:
Never mind. This is just in tests, so I don't care.